FungibleToken.ts

```diff
@@ -22,20 +22,21 @@ import {
 } from "o1js"
 import { FungibleTokenAdmin, FungibleTokenAdminBase } from "./FungibleTokenAdmin.js"
 
-export interface FungibleTokenDeployProps extends Exclude<DeployArgs, undefined> {
+interface FungibleTokenDeployProps extends Exclude<DeployArgs, undefined> {
```

Comment:
Why not expose this?

Reply:
We didn't need to export it for any of the tests or examples. But if you think it's useful to export it, we can do that as well.